### PR TITLE
governance-model: initial governance model

### PR DESCRIPTION
**Pull request by nalla**
This is the initial checkin of the governance model on which this project
is ran. Edit the `governance-model.md` to edit the model. Note that this
is in no way a final model. Its a proposal that should be worked on.

See: git-for-windows/git#110
Signed-off-by: 마누엘 <nalla@hamal.uberspace.de>

### CODE REVIEW

**Comment by dscho** [comment]
This is excellent! And actually, I think you managed to guide my thinking a bit, too, especially about the users' role in this project.

**Comment by dscho** [comment]
I just had a look at this locally... The only qualm I have about it is that the layout is a bit off due to the "governance model" link in the top bar. I would like to make it less prominent, because it is really less important than the FAQ, the repository or the mailing list.

How about putting a link into the repository's README.md file, as well as into the contributing section?

File: index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
 			<nav class="content">
 				<a href="index.html"><h1 class="gittext lowercase">Git<span>for Windows</span></h1></a>
 				<ul class="list-unstyled">
+					<li><a href="governance-model.html">Governance Model</a></li>
```
Comment (nalla) [inline index.html]: @dscho What about removing this and just put it in the footer then?

**Comment by nalla** [comment]
> How about putting a link into the repository's README.md file, as well as into the contributing section?

I presume you mean [git-for-windows/git/README.md](https://github.com/git-for-windows/git/blob/master/README.md) and [git-for-windows/git/wiki/How-to-participate](https://github.com/git-for-windows/git/wiki/How-to-participate)?

**Comment by dscho** [comment]
> I presume you mean git-for-windows/git/README.md and git-for-windows/git/wiki/How-to-participate?

Good idea, but maybe also on the front page, near the issue tracker reference?

File: index.html (hunk 1 above)
Comment (dscho) [inline index.html]: When I resized my browser window to sizes close to a typical tablet, the footer looked more pleasant than the header, but I'd prefer to keep them identical, for visual pleasure.

**Comment by nalla** [comment]
In 2b9ef90 I removed the links from the navigation and added a paragraph for the governance model.

**Comment by dscho** [comment]
Perfect!

**Pull request merged**